### DISCUSSION TITLE
Awesome Type Inference Resources

### DISCUSSION THREAD

**Discussion by mutouyuguo** [deeplang-org/deeplang-type-system#2] [General]
- https://www.youtube.com/watch?v=45rZ1dYkgbY
- https://www.youtube.com/channel/UCfRo7FqOTUB2lZe-Dodzi6g/videos

**Comment by chinesebear** [comment]
https://opam.ocaml.org/blog/about-utop/

**Comment by mutouyuguo** [comment]
https://xiongyingfei.github.io/DPPL/2021/main.htm
